src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -15,8 +19,21 @@
     {
         <script asp-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.min.js" debug-src="~/OrchardCore.Lists/Scripts/ci-list-ordering.js" asp-name="ci-list-ordering" at="Foot" depends-on="sortable, jQuery"></script>
     }
-}
 
+    var authorizedIndexes =
+        await Task.WhenAll(
```

Comment:
Don't use `WhenAll()` since it results in parallelized code but we can't guarantee everything is thread-safe here. Just await them sequentially.